.teamcity/_self/projects/WebApp.kt

```diff
@@ -788,6 +789,44 @@ object PreReleaseE2ETests : E2EBuildType(
 	}
 )
 
+object AuthenticationE2ETests : E2EBuildType(
+	buildId = "Calypso_E2E_Authentication",
+	buildUuid = "f5036e29-f400-49ea-b5c5-4aba9307c5e8",
+	buildName = "Authentication E2E Tests",
+	buildDescription = "Runs a suite of Authentication E2E tests.",
+	concurrentBuilds = 1,
+	testGroup = "authentication",
+	buildParams = {
+		param("env.VIEWPORT_NAME", "desktop")
+	},
+	buildFeatures = {
+		notifications {
+			notifierSettings = slackNotifier {
+				connection = "PROJECT_EXT_11"
+				sendTo = "#e2eflowtesting-notif"
+				messageFormat = verboseMessageFormat {
+					addStatusText = true
+				}
+			}
+			branchFilter = "+:<default>"
+			buildFailedToStart = true
+			buildFailed = true
+			buildFinishedSuccessfully = false
```

Comment:
Normally we'd set this to `true` so that we're notified for successful builds, but I think doing so is detrimental:
- it can lead individuals to think the failure is flakey.
- it pollutes the channel scrollback.